lib/elixir/lib/enum.ex

```diff
@@ -2435,15 +2434,16 @@ defmodule Enum do
 
   ## uniq
 
-  defp do_uniq([h|t], acc, fun) do
+  defp do_uniq([h|t], set, fun) do
```

Comment:
I have mixed feeling about this. I consider accumulator something I would return later on, and this is not returned, but if that's important I will change it.